Source/MonoGame.Extended/Maps/Tiled/TiledObject.cs

```diff
@@ -23,11 +31,9 @@ public TiledObject(TiledObjectType objectType, int id, int? gid, float x, float
         public int? Gid { get; }
         public TiledObjectType ObjectType { get; }
         public string Name { get; set; }
-        public float X { get; }
-        public float Y { get; }
         public float Width { get; }
         public float Height { get; }
-        public Vector2 Position => new Vector2(X, Y);
+        public Vector2 Position { get; private set; }
```

Comment:
I'm a bit new when it comes to C# properties and what not, so I though that you needed private set at all times to change the value. Being able to change it in the constructor does make sense now that I think about it though...